lib/twitter-ads/account.rb

```diff
@@ -94,6 +94,46 @@ def features
       response.body[:data]
     end
 
+    # Returns a collection of media creatives available to the current account.
+    #
+    # @param id [String] The MediaCreative ID value.
+    # @param opts [Hash] A Hash of extended options.
+    # @option opts [String] :line_item_id Scope the result to a line item ID.
+    # @option opts [String] :campaign_id Scope the result to a campaign ID.
+    # @option opts [Boolean] :with_deleted Indicates if deleted items should be included.
+    # @option opts [String] :sort_by The object param to sort the API response by.
+    #
+    # @return A Cursor or object instance.
+    def media_creatives(id = nil, opts = {})
+      load_resource(Creative::MediaCreative, id, opts)
+    end
+
+    # Returns a collection of account media available to the current account.
+    #
+    # @param id [String] The Account Media ID value.
+    # @param opts [Hash] A Hash of extended options.
+    # @option opts [String] :account_media_ids Comma separated account media ids
+    # @option opts [Boolean] :with_deleted Indicates if deleted items should be included.
+    # @option opts [String] :sort_by The object param to sort the API response by.
+    #
+    # @return A Cursor or object instance.
+    def account_media(id = nil, opts = {})
+      load_resource(Creative::AccountMedia, id, opts)
+    end
+
+    # Returns a collection of videos available to the current account.
+    #
+    # @param id [String] The Video ID value.
+    # @param opts [Hash] A Hash of extended options.
+    # @option opts [String] :video_ids Comma separated video ids
+    # @option opts [Boolean] :with_deleted Indicates if deleted items should be included.
+    # @option opts [String] :sort_by The object param to sort the API response by.
+    #
+    # @return A Cursor or object instance.
```

Comment:
This endpoint has been deprecated in v4 (i.e., the `:account_id/videos` endpoint). Feel free to remove this code block entirely.